app/helpers/govuk_design_system_helper.rb

```diff
@@ -21,8 +21,28 @@ def govuk_breadcrumb_builder
     GdsDesignSystemBreadcrumbBuilder
   end
 
+  def govuk_detail(summary_text = nil, open = false, tag_options = {}, &block)
```

Comment:
This is great. In order to be consistent we should use this for the govuk_details I added as part of unlinking without a maat id (although not as part of this pr).